CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+
+Please keep this workflow in mind as you read through the document.
+
+### Communication Channels
+
+- Email: [operator-framework-olm-dev](mailto:operator-framework-olm-dev@googlegroups.com)
+- Slack: [#olm-dev](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [olm-gg](https://groups.google.com/g/operator-framework-olm-dev)
+- Weekly in Person Working Group Meeting: [olm-wg](https://github.com/operator-framework/community#operator-lifecycle-manager-working-group) 
+
+## How are Milestones Designed?
+
+It's unreasonable to attempt to consider all of the design requirements laid out in the [OLM V1 PRD](https://docs.google.com/document/d/1-vsZ2dAODNfoHb7Nf0fbYeKDF7DUqEzS9HqgeMCvbDs/edit) from the onset of the project. Instead, the community attempts to design Milestones with the following principles:
+- Milestones are tightly scoped units of work, ideally lasting one to three weeks.
+- Milestones are derived from the OLM V1 PRD.
+- Milestones are "demo driven", meaning that a set of acceptance criteria is defined upfront and the milestone is done as soon as some member of the community can run the demo.
+- Edge cases found during development are captured in issues and assigned to the GA Milestone, which contains a list of issues that block the release of operator-controller v1.0.0 .
+
+This "demo driven" development model will allow us to collect user experience and regularly course correct based on user feedback. Subsequent milestone may revert features or change the user experience based on community feedback.
+
+The project maintainer will create a [GitHub Discussion](github.com/operator-framework/operator-controller/discussions) for the upcoming milestone once we've finalized the current milestone. Please feel encouraged to contribute suggestions for the milestone in the discussion.
+
+## Where are Operator Controller Milestones?
+
+Ongoing or previous Operator-Controller milestones can always be found in the [milestone section of our GitHub Repo](https://github.com/operator-framework/operator-controller/milestones).
+
+### How are Subproject Issues Tracked?
+
+As discussed earlier, the operator-controller adheres to a microservice architecture, where multiple projects contribute to the overall experience. As such, when designing an operator-controller milestone, the community may need to file an issue against RukPak, Deppy, or Catalogd. Unfortunately, the operator-controller milestone cannot contain issues from one of its subprojects. As such, we've introduced the concept of a "Dependency Issue", described below:
+
+> Dependency Issues: An issue tracked in a milestone that "points" to an issue in another project with a URL.
+
+## Submitting Issues
+
+Unsure where to submit an issue? 
+- The Operator-Controller project
+- The RukPak project
+- The Deppy project
+- The Catalogd project
```

Comment:
Good question, I added similar links and descriptions earlier in the doc, perhaps making that a section and linking it here is the right idea?

Reply:
Ah I see, I don't think it hurts to be extra verbose in case that section is missed or someone is going through and looking at all the links and forgets that information was already provided. I think there is quite a bit of information to digest when it comes to OLMv1 so being that bit more verbose and having *some* repetition might be beneficial

Reply:
Addressed in the latest commit.